BenadrylAutomation/testng.xml

```diff
@@ -4,13 +4,14 @@
 	<listeners>
 		<listener class-name="com.jnj.utility.CustomListener" />
 	</listeners>
-
-	<test name="Regression Suite Testcases">
+ 	<test name="Regression Suite Testcases">
 		<classes>
 			<class name="com.jnj.testcases.HomePageTest" />
 			<class name="com.jnj.testcases.SignUpPageTest" />
 			<class name="com.jnj.testcases.HeaderPageTest" />
 			<class name="com.jnj.testcases.DifferencePageTest" />
+			<class name="com.jnj.testcases.ComparePageTest" /> 
```

Comment:
This is a duplicate entry @RashiTiwari21 . Please correct this

Reply:
fixed

Reply:
@RashiTiwari21 Header test is missing from testng file. Please recheck

Reply:
fixed

Reply:
Verified @RashiTiwari21